src/blackbox_render.c

```diff
@@ -326,10 +347,25 @@ void updateFieldMetadata()
 void drawCommandSticks(int64_t *frame, int imageWidth, int imageHeight, cairo_t *cr)
 {
     double rcCommand[4] = {0, 0, 0, 0};
-    const int stickSurroundRadius = imageHeight / 11, stickSpacing = stickSurroundRadius * 3;
+    int stickSurroundRadius = imageHeight / 11;
+    if(options.sticksWidth > 0) {
```

Comment:
Can you please add spaces after keywords? In general, the coding style rules from https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md should be applied wherever possible.

Reply:
As suggested spaces have been added to comply with the coding guidelines.